packages/editor/src/editor/sidebar/Properties.tsx

```diff
@@ -51,6 +54,8 @@ const PropertySection = ({ section, fields }: { section: string; fields: Visible
 
 const PropertySubSection = ({ title, fields }: { title: string; fields: VisibleFields }) => {
   const { element, setElement } = useData();
+  const { context } = useAppContext();
+  const variableInfo = useMeta('meta/data/attributes', context, { types: {}, variables: [] }).data;
```

Comment:
now every property sub section is a consumer of this meta data even if it needs it or not!
This things here should be done in a generic field.